src/algebra/Module/basic.lean

```diff
@@ -0,0 +1,54 @@
+/-
+Copyright (c) 2019 Robert A. Spencer. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Robert A. Spencer
+-/
+import algebra.module
+import algebra.punit_instances
+import category_theory.types
+import linear_algebra.basic
+open category_theory
+
```

Comment:
Could you add file-level documentation as described at https://github.com/leanprover-community/mathlib/blob/master/docs/contribute/doc.md ?